src/bioregistry/data/curated_mappings.sssom.tsv

```diff
@@ -10,40 +11,80 @@ bioregistry:afo	Not	skos:exactMatch	agroportal:AFO	orcid:0000-0003-4423-4370	sem
 bioregistry:afo	Not	skos:exactMatch	fairsharing:FAIRsharing.595710	orcid:0000-0001-9439-5346	semapv:MappingReview	
 bioregistry:agricola	Not	skos:exactMatch	lov:agr	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
 bioregistry:agricola	Not	skos:exactMatch	uniprot:DB-0266	orcid:0000-0001-9439-5346	semapv:MappingReview	
+bioregistry:apollosv		skos:exactMatch	ontobee:APOLLO_SV	orcid:0009-0009-5240-7463	semapv:MappingReview	
 bioregistry:ark	Not	skos:exactMatch	bartoc:20297	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
 bioregistry:arxiv	Not	skos:exactMatch	bartoc:20434	orcid:0000-0001-9439-5346	semapv:MappingReview	
+bioregistry:astd	Not	skos:exactMatch	prefixcommons:astd	orcid:0009-0009-5240-7463	semapv:MappingReview	
 bioregistry:atc	Not	skos:exactMatch	agroportal:ATC	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
 bioregistry:atc	Not	skos:exactMatch	hl7:2.16.840.1.113883.6.77	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
+bioregistry:atc	Not	skos:exactMatch	togoid:Atc	orcid:0009-0009-5240-7463	semapv:MappingReview	
```

Comment:
Just spot checking some of these, this one I think is a correct mapping, @nagutm do you recall why you made the choice for this to be considered negative?

Reply:
```suggestion
bioregistry:atc		skos:exactMatch	togoid:Atc	orcid:0009-0009-5240-7463	semapv:MappingReview	
```